docs/user/alerting/action-types.asciidoc

```diff
@@ -27,6 +27,10 @@ a| <<slack-action-type, Slack>>
 
 | Send a message to a Slack channel or user.
 
+a| <<servicenow-action-type, ServiceNow>>
+
+| Create new ServiceNow incident.
```

Comment:
The action types list is in alphabetical order so ServiceNow should go below ServerLog.